README.md

```diff
@@ -70,6 +70,13 @@ The directives have the following options:
     Button()
 ```
 
+### Configuration
+
+You conf.py has two extra configuration options:
+
+ * jupyter_sphinx_require_url: url for `require.js` (if your theme already provides this, set it to False or '')
```

Comment:
What about:
 * jupyter_sphinx_embed_url: if not set will be chosen based on next config value (jupyter_sphinx_embed_with_require)
 * jupyter_sphinx_embed_with_require: True/False will fill in jupyter_sphinx_embed_url if not set
* jupyter_sphinx_require_url: url to require.js, or empty to not include it.